scripts/hf_upload.py

```diff
@@ -0,0 +1,18 @@
+import os
+from datetime import datetime
+from huggingface_hub import login, HfFileSystem
+import forecast_csv
+
+if __name__ == "__main__":
+
+    hf_token = os.getenv("HF_TOKEN")
+    hf_repo = os.getenv("HF_REPO")
+
+    now = datetime.utcnow()
+    forecast = forecast_csv.forecast_for_site(latitude=51.75, longitude=-1.25, capacity_kwp=1.25, init_time=now)
```

Comment:
@zakwatts do you know any good sites we could run this forecast for? Where we have reliable live data too, so we can test against it

Reply:
I'll have a look, did you get round to processing the more up to data household dataset? could use that or could do a quick analysis of the sites from the test set we made, check they have good data in the production siteSQL database and go from there?

Reply:
I found one that looks reliable. It's at `51.59`, `-1.89` with a capacity of 4 kw.  @mduffin95 would you be able to update the run to these parameters?

Reply:
@mduffin95 do you mind just updating to use these parameters, and then I think we are good to merge